environment.py

```diff
@@ -526,5 +526,13 @@ def env() -> Mapping[str, Optional[str]]:
         # testing feature branches in GitLab before they are merged to the
         # develop branch, 0 otherwise. Personal deployments have this set to 0.
         #
-        'AZUL_IS_SANDBOX': '0'
+        'AZUL_IS_SANDBOX': '0',
+
+        # The UCSC-GI organization's Slack workspace ID.
+        'azul_slack_workspace_id': 'T09P9H91S',
+
+        # The #team-boardwalk channel ID which is the main communications
+        # channel for Azul, and the location where we wish to receive Azul
+        # related alarms and notifications.
+        'azul_slack_team_boardwalk_channel_id': 'C705Y6G9Z'
```

Comment:
```suggestion
        # The #team-boardwalk channel ID which is the main communications
        # channel for Azul, and the location where we wish to receive Azul
        # related alarms and notifications.
        'azul_slack_team_boardwalk_channel_id': 'C705Y6G9Z'
```

You shouldn't bake the channel name into either the documentation or the variable name.